api/tests/unit/config/fixtures/data_fixtures/AppFixturesTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\unit\config\fixtures\data_fixtures;
+
+use App\DataFixtures\AppFixtures;
+use App\Entity\Question;
+use App\Entity\Quiz;
+use PHPUnit\Framework\TestCase;
+
+final class AppFixturesTest extends TestCase
+{
+
+    private $appFixtures;
+    private $objectManager;
+
+    protected function setUp(): void
+    {
+        $objectManager = $this->getMockBuilder('Doctrine\Persistence\ObjectManager')
+            ->onlyMethods(['flush', 'persist'])
+            ->getMockForAbstractClass();
+        $this->objectManager = $objectManager;
+        
+        $appFixtures = new AppFixtures;
+        $appFixtures = $this->invokeProperty($appFixtures, 'objectManager', $objectManager);
+        $this->appFixtures = $appFixtures;
+    }
+
+    public function invokeProperty(object &$object, string $propertyName, mixed $parameter): object
+    {
+        $reflection = new \ReflectionClass(get_class($object));
+        $property = $reflection->getProperty($propertyName);
+        $property->setAccessible(true);
+        $property->setValue($object, $parameter);
+
+        return $object;
+    }
+
+    public function testLoad()
+    {
+        $loadFunction = $this->appFixtures->load($this->objectManager);
+
+        self::assertNull($loadFunction);
+    }
+
+    public function testCreateQuizzes()
+    {
+        $createQuizzesFunction = $this->appFixtures->createQuizzes();
```

Comment:
Can we Return the quizzes and check that they are the correct instances instead of checking for null.